express-notes-api/index.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+require('./data.json');
+
+app.get('/api/notes/', (req, res) => {
+
+  const notesArray = [];
+
+  fs.readFile('./data.json', 'utf8', (err, data) => {
```

Comment:
Instead of using `readFile` in every endpoint you can just `require` the data at the top of the file:

```JAVASCRIPT
const data = require('./data.json');
```